### ISSUE TITLE
Error shown multiple times in "Search by location" without location given

### ISSUE THREAD

**Issue by Gerwanese** [BeWelcome/rox#216]
How to reproduce:
1. Go to "Find members" => "Search by location"
2. Don't enter anything, just hit "Find Members"
![grafik](https://user-images.githubusercontent.com/17579552/105650612-88f7d680-5eb4-11eb-814d-a25e78d3eafb.png)

I would expect to get only the first line of error instead of all three; at least first and third are redundant.
Obviously this is a very minor issue and the error message is still clear in telling me what to do so you might choose not to fix it.

**Comment by thisismeonmounteverest** [comment]
@Gerwanese I agree that's overkill. I'd still go for two lines (or maybe combine the two) as it often happens that people are to quick to click on the search button while they should have waited a bit to select from the drop-down.

**Comment by crumbking** [comment]
I suggest one line: Please enter a location and select from the appearing list after entering a location.
Please no !
And maybe a yellow toast will be enough. ;-)
